zip-0200.rst

```diff
@@ -103,7 +103,7 @@ ACTIVATION_HEIGHT
 
   - As of the time of writing (the 1.0.15 release), the release cycle is six weeks long, and nodes undergo
     End-of-Service halt 16 weeks after release. Thus, if version ``X`` contains support for a network upgrade,
-    version ``X-1`` will be deprecated 10 weeks after the release of version ``X``, which is about 2.3 months.
+    version ``X-1`` will be deprecated 10 weeks after the release of version ``X``, which is about 2.5 months.
```

Comment:
It is actually closer to 2.3. (70 days divided by the average length of a month.)

Reply:
True!
I made the mistake of assuming that two weeks are approximately half a month. It's a pity humans didn't come up with decimally related units for measuring time. 2.3 would be easier to interpret then.